modules/AOW_WorkFlow/AOW_WorkFlow.php

```diff
@@ -75,6 +80,8 @@ class AOW_WorkFlow extends Basic
      */
     private function getSQLOperator($key)
     {
+        global $timedate;
```

Comment:
do not use superglobals or globals if it's not necessary, pass it as an (optional) dependency.